modules/subchannel/examples/EBR-II/plot_results.py

```diff
@@ -0,0 +1,150 @@
+# %%
```

Comment:
Is this file needed? This is a python file for plotting. I am good to add it, but please explain the function of this file in the header to avoid confusions